src/components/tags/tags.js

```diff
@@ -111,8 +127,39 @@ export class TagsComponent {
     );
   }
 
-  handleDeleteClick() {
-    console.log("handleDeleteClick");
+  handleDeleteClick(event) {
+    let moreButton = event.target.closest(".button-more");
+    this.listItem = moreButton.closest(".tags__list-item");
```

Comment:
write it in one row: `this.tagToDel = event.target.closest(".tags__list-item");` (listItem = tagToDel)